frontend/package.json

```diff
@@ -1,60 +1,59 @@
 {
-  "name": "dashboard",
-  "version": "0.0.1",
+  "name": "sa-dashboard",
```

Comment:
Please set this to `"atlas-of-opportunity"`. Atlas v2 is going to be used in more places than v1, so I'd like to reduce references to South Australia in the project's name.